jobs/tendrl-package-validation-jobs.yml

```diff
@@ -111,90 +208,92 @@
     block-downstream: true
     block-upstream: true
     wrappers:
-        - ansicolor:
-            colormap: "xterm"
+      - ansicolor:
+          colormap: "xterm"
     node: '{node}'
     builders:
-        - shell: |
-            #!/bin/bash -x
-            #cd usmqe-tests
-            ## prepare hosts file
-            qe_server=$(grep -A1 '\[qe_server\]' ${{HOME}}/{clustername}.hosts | tail -1)
-            scp ${{HOME}}/{clustername}.hosts usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/conf/usm.hosts
-            # prepare required variables
-            echo "" > usm.ini
-            echo "[usmqepytest]" >> usm.ini
-            echo "usm_log_level = DEBUG" >> usm.ini
-            echo "usm_rpm_baseurl = ${{USM_REPO}}" >> usm.ini
-            echo "usm_rpm_gpgkey_url = ${{USM_REPO_GPGKEY}}" >> usm.ini
-            scp usm.ini usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/conf/usm.ini
-            test="/home/usmqe/usmqe-tests/usmqe_tests/rpm/test_rpm.py";
-            # prepare env
-            ssh usmqe@${{qe_server}} "export PATH=\$PATH:\$HOME/.local/bin:\$HOME/bin; \
-              scl enable rh-python35 'pip install --user -r /home/usmqe/usmqe-tests/requirements.txt' ; \
-              source /opt/rh/rh-python35/enable ; \
-              export X_SCLS=\$(scl enable rh-python35 'echo $X_SCLS') ; \
-              env ; \
-              cd /home/usmqe/usmqe-tests/; \
-              cat conf/usm.hosts ; \
-              cat conf/usm.ini ; \
-              echo py.test --color=yes --junit-xml=logs/result.xml -vvvrx ${{test}}; \
-              py.test --color=yes --junit-xml=logs/result.xml -vvvrx ${{test}}"
-            pwd
-            echo "qe_server=${{qe_server}}"
-            echo "scp -r usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/logs ."
-            scp -r usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/logs .
-            sed -i 's/#x1B\[[01]\(;[0-9][0-9]\)\?m//g' logs/result.xml
-        - shell: |
-            #!/bin/bash -x
-            #cat ${{HOME}}/{clustername}.hosts
+      - shell: |
+          #!/bin/bash -x
+          #cd usmqe-tests
+          ## prepare hosts file
+          qe_server=$(grep -A1 '\[qe_server\]' ${{HOME}}/{clustername}.hosts | tail -1)
+          scp ${{HOME}}/{clustername}.hosts \
+            usmqe@${{qe_server}}:/home/usmqe/usmqe-tests/conf/usm.hosts
+          # prepare required variables
+          echo "" > usm.ini
```

Comment:
Ok, agreement is that this pull request will be merged and then there will new pull request for usm.ini changes